src/bm_sim/actions.cpp

```diff
@@ -289,12 +291,18 @@ ActionFnEntry::execute(Packet *pkt) const {
 
   auto &primitives = action_fn->primitives;
   size_t param_offset = 0;
+  size_t si_offset = 0;
   // primitives is a vector of pointers
   BMLOG_TRACE_SI_PKT(*pkt, action_fn->get_source_info(),
-                     "Executing action {}", action_fn->get_name());
+                     "Action {}", action_fn->get_name());
   for (auto primitive : primitives) {
+    BMLOG_TRACE_SI_PKT(*pkt, action_fn->source_infos[si_offset],
```

Comment:
@antoninbas Any thoughts on my questions about possible changes to the default log level, or creating a new log level more detailed than the current '-L trace' ?